grails-app/services/org/bbop/apollo/SequenceService.groovy

```diff
@@ -39,22 +39,19 @@ class SequenceService {
      * @param feature
      * @return
      */
-    String getResiduesFromFeature(Feature feature ) {
-        List<FeatureLocation> featureLocationList = FeatureLocation.createCriteria().list {
-            eq("feature",feature)
-            order("fmin","asc")
+    String getResiduesFromFeature(Feature feature) {
+        String returnResidues = ""
+        def orderedFeatureLocations = feature.featureLocations.sort { it.fmin }
```

Comment:
nice, doing it memory vs a separate sql query will be much faster, esp. if already there via a join